### DISCUSSION TITLE
New features

### DISCUSSION THREAD

**Discussion by djokenzi** [bangcorrupt/hacktribe#174] [Ideas]
Hi guys,

Great job on hacking the Electribe!

I'm going to list some features I'd like to see on the Hacktribe. I'm aware that it might not be possible or very difficult to implement them, but some of you might find them useful:

1) 8 Bar patterns
2) Sidechaining for each part
3) A way to have control of how much of the signal you're sending to master effect, for each part. For example, by holding MFX Send button and turning the Volume knob to adjust the send amount, rather than just having On/Off control. 
4) More than one insert effect. Probably difficult to achieve and would eat up voices very quickly.
5) Reverbs as insert FX. Would like to see new types of reverbs on the Hacktribe, at least on the MFX section. Reberbs similar to those on the NTS-1 would be great. 

Thanks in advance for considering the above features. Keep the Electribe updated! :)

**Comment by bangcorrupt** [comment]
1. [FAQ](https://github.com/bangcorrupt/hacktribe/wiki/FAQ#8-bar-patterns)
2. (4. 5.) #75. Reverb as an insert is unlikely, it uses a lot of resources.  Greater control of the reverb parameters is coming, possibly more reverb algorithms in the future.  
3. I think this will be possible when FX editing is complete. #157.

**Comment by djokenzi** [comment]
Speaking of having control of how much of the signal you're sending to master effect for each part, I just had another idea. 

It would be great if there was an EQ or a high pass filter also, so you could send only certain frequencies to the MFX. That would be useful to enrich a sound of a bassline that has upper harmonics as well, by only sending mids and highs to the reverb, and leave the lows dry to avoid  muddying up the sound. This could free up a part in case you're using more than one part to layer sounds.

**Reply by bangcorrupt** [reply]
The reverb has high and low EQ built in, there is only control of level, not frequency.  For high pass filtering you could copy the part, filter it, then send it to a 100% wet reverb.
